sql/core/src/main/scala/org/apache/spark/sql/DataFrame.scala

```diff
@@ -730,6 +731,23 @@ class DataFrame private[sql](
     Generate(generator, join = true, outer = false, None, logicalPlan)
   }
 
+  /**
+   * Returns a new [[DataFrame]] that omits rows with `NULL` values in the specified columns.
+   * If no columns are specified, a row is omitted if any column contains `NULL`.
+   *
+   * @group dfops
+   */
+  @scala.annotation.varargs
+  def dropna(cols: String*): DataFrame = {
```

Comment:
Is it possible to do what cheng said? I think dropNull is a much better name, but if we have an alias dropna, I think it would be good for pandas compatibility. In general I think aliasing things might necessary in the way we approach data frames since we're trying to meet a few different requirements.